test/cp.js

```diff
@@ -112,6 +112,32 @@ test('dest already exists', t => {
   t.is(shell.cat('resources/file2').toString(), oldContents);
 });
 
+test('-nR does not overwrite an existing file at the destination', t => {
+  // Create tmp/new/cp/a
+  const dest = `${t.context.tmp}/new/cp`;
```

Comment:
Yup, looks like we caught a bug. We should probably be converting the output to use a `/` instead of a `\`.